src/Components/CreateImageWizard/steps/FileSystem/FileSystemConfiguration.tsx

```diff
@@ -74,9 +79,34 @@ const FileSystemConfiguration = () => {
       <TextContent>
         <Text>
           Create partitions for your image by defining mount points and minimum
-          sizes. Image builder creates partitions with a logical volume (LVM)
-          device type.
+          sizes.
         </Text>
+        <FormGroup>
+          <Radio
+            id="raw partitioning radio"
+            ouiaId="raw-partitioning-radio"
+            label="Use raw partitioning"
+            name="fsc-radio-raw"
+            description="Filesystems will be created directly on top of physical partitions"
```

Comment:
I believe this was feedback on the UX call.

```suggestion
            description="File systems will be created directly on top of physical partitions"
```